modules/virt-deleting-deployment-custom-resource.adoc

```diff
@@ -5,7 +5,7 @@
 //This file contains UI elements and/or package names that need to be updated.
 
 [id="virt-deleting-deployment-custom-resource_{context}"]
-= Deleting the {VirtProductName} Operator Deployment custom resource
+= Deleting the {VirtProductName} Operator deployment custom resource
```

Comment:
In this case, "Deployment" is capitalized because that's how it appeared in the UI

Reply:
Thanks for the catch :)